bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="visualcrossing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Visual Crossing Binding</name>
+	<description>
+		Visual Crossing Weather is the easiest-to-use and lowest-cost source for historical and forecast weather
+		data. Our Weather API (https://www.visualcrossing.com/weather-api) is designed to integrate easily into any app or
+		code, and our prices are lower than any other provider in the industry.
```

Comment:
Same comment as in the readme. Please remove the our and we parts as this is not written from an openHAB point of view.

Reply:
done